airtouch4pyapi/communicate.py

```diff
@@ -70,6 +70,10 @@ async def SendMessagePacketToAirtouch(messageString, ipAddress):
         data = response
     except socket_error as serr:
         data = serr;
+    
```

Comment:
@sibowler have you replicated the issue regarding the laggy controller tablet and verified this fixes it? if not id rather split this out and look at the bug separately - otherwise this pr looks good

Reply:
@sibowler Managed to replicate the issue and verify that closing the connection does fix it - so will merge this PR in including this fix - Thanks very much!

Reply:
Hey @LonePurpleWolf  - sorry for not replying on this one. Was away for a week on holiday, but glad the connection closing sorted it out.